packages/server/src/queries/complex/assets/bridge.ts

```diff
@@ -0,0 +1,30 @@
+import { Asset as AssetListAsset, AssetList } from "@osmosis-labs/types";
+
+import { Asset } from ".";
+
+/** A bridgeable asset. */
+export type BridgeAsset = {
+  transferMethods: AssetListAsset["transferMethods"];
+  counterparty: AssetListAsset["counterparty"];
+};
+
+/** Appends bridge info to a given asset. If asset is not found in asset list, empty bridge info will be returned.
+ *  @throws if a given asset is not found in asset list.
+ */
+export function getBridgeAsset<TAsset extends Asset>(
```

Comment:
What are your thoughts on including the `transferMethod` and `counterparty` to our `getAssetFromAssetList` instead of adding a new method?

Reply:
Also, to clarify, are you suggesting removing this function altogether? Or just using `getAssetFromAssetList` within it?

Reply:
I'm now suggesting we include the `getAssetFromAssetList` within